### PR TITLE
Some more user service db refactorings

### PR DESCRIPTION
**Pull request by mrcage**

### CODE REVIEW

File: htdocs/admin/content/galaxy/exploration.inc.php [hunk 1]
```diff
@@ -30,37 +42,32 @@
...
+        $sy = $request->request->getInt('sy');
+        $cx = $request->request->getInt('cx');
+        $cy = $request->request->getInt('cy');
+        $radius = abs($request->request->getInt('radius'));
 
         $cell = $cellRepository->getCellIdByCoordinates($sx, $sy, $cx, $cy);
         if ($cell !== null) {
             [$absX, $absY] = $cell->getAbsoluteCoordinates($config->param1Int('num_of_cells'), $config->param2Int('num_of_cells'));
-            $user->setDiscovered($absX, $absY, $radius);
+            $userUniverseDiscoveryService->setDiscovered($uid, $absX, $absY, $radius);
```
Comment (glaubinix) [inline htdocs/admin/content/galaxy/exploration.inc.php:55]: Might almost be easier to pass in `$cell` then we don't need the config service here

File: src/User/UserUniverseDiscoveryService.php [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+        return $discoveryMask;
+    }
+
+    public function discovered(int $userId, int $absX, int $absY): bool
+    {
+        $sy_num = $this->config->param2Int('num_of_sectors');
+        $cy_num = $this->config->param2Int('num_of_cells');
+
+        $mask = $this->loadDiscoveryMask($userId);
```
Comment (glaubinix) [inline src/User/UserUniverseDiscoveryService.php:45]: This now fetches the discovery mask for every field. Wonder whether instead we should pass in a User object or some sort of context

File: htdocs/admin/content/galaxy/exploration.inc.php (hunk 1 above)
Comment (mrcage) [reply htdocs/admin/content/galaxy/exploration.inc.php:55]: the "cell" does not know its absolute coordinates, as this depends on configuration....

File: src/User/UserUniverseDiscoveryService.php (hunk 2 above)
Comment (mrcage) [reply src/User/UserUniverseDiscoveryService.php]: done 
6577bec

File: htdocs/admin/content/galaxy/exploration.inc.php (hunk 1 above)
Comment (mrcage) [reply htdocs/admin/content/galaxy/exploration.inc.php]: done 
eaf3ebd

**Pull request merged**